src/Reducer.ts

```diff
@@ -7,7 +7,7 @@ function createInitialState(): Types.IMsalState {
     return {};
 }
 
-export function msalReducer(state = createInitialState(), action: Action): Types.IMsalState {
+export function msalReducer(state: Types.IMsalState | undefined, action: Action): Types.IMsalState {
```

Comment:
Is this a readability change or a required change by the TypeScript upgrade?

Reply:
It's caused by redux types upgrade I think. See https://github.com/reduxjs/redux/blob/6bf721e37475ad9d3f710251c4119ae30f9327f8/index.d.ts#L59